RecoLocalCalo/HcalRecProducers/python/HFQIE10Reconstructor_cfi.py

```diff
@@ -0,0 +1,15 @@
+import FWCore.ParameterSet.Config as cms
+
+hfQIE10Reco = cms.EDProducer("HcalSimpleReconstructor",
```

Comment:
please keep the name of the module and the cfi file name the same.
hfQIE10Reco_cfi